thavalon-webapp/src/App.tsx

```diff
@@ -61,8 +71,10 @@ function App() {
           <Logout setLoggedIn={() => setLoggedIn(false)} />
         </Route>
         <Route path="/register" render={() => registerPage()} />
+        <Route path="/game"></Route>
       </Switch>
       {showLoginModal && <Login setLoggedIn={setLoggedIn} setShowLoginModal={setShowLoginModal} showLoginModal={showLoginModal} />}
+      <CreateJoinGameModal show={showCreateJoinGameModal} setOpen={setShowCreateJoinGameModal} />
```

Comment:
Is it possible to update Login to show similarly? Or this show a react-bootstrap thing?

Reply:
It should be. The modal I used is your modal, so I don't think react-bootstrap is involved. Did you want me to update in this PR?

Reply:
Sure, if it's easy enough to do